src/introduction/abtnode-setup/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: 'ABT Node Setup'
+description: 'ABT Node Setup'
+keywords: 'abtnode, blocklet'
+author: 'tigeryu8900'
+category: ''
+layout: 'documentation'
+tags:
+  - 'abtnode'
+---
+ABT Node can be installed in the following ways :
+
+  - [Use the binary distribution](#use-the-binary-distribution)
+  - [Use the official Docker image](#use-the-official-docker-image)
+
+You can use either of the above ways to deploy it on any infrastructure like Cloud, Virtual machines, workstations, etc.
+
+> If you are interested in running ABT Node locally on your device, we currently support [Linux] and [macOS]. If you are using another OS like Windows, we recommend using Windows Subsystem Linux (WSL), or you can use a virtual machine and make sure your ABT Wallet can access your VM's IP address.
+
+## Use the binary distribution
+
+ABT Node is a [NodeJS application](https://www.npmjs.com/package/@abtnode/cli). You must have NodeJS Runtime and Node Package Manager (NPM) to install it.
+
+### Environment Setup
+You need to install NodeJS and NPM for installing ABT Node binary. ArcBlock recommends using Node Version Manager(NVM) for [installing NodeJS](https://nodejs.org/en/download/package-manager/#nvm).
+
+#### Step 1: Install [NVM]
+
+Install [NVM] using the below command:
+
+```bash
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+```
+
+<details>
+<summary>Sample output</summary>
+
+![install nvm](./images/install_nvm.gif)
+
+</details>
+
+#### Step 2: Install [Node.js] and [npm]
+
+Get the latest LTS version of [Node.js] using below command (you can list all versions with `nvm ls-remote`):
+
+```bash
+nvm ls-remote | grep 'Latest LTS'
+```
+
+<details>
+<summary>Sample output</summary>
+
+![list Node.js versions](./images/list_node_versions.gif)
+
+</details>
+
+At the time of writing this document, the latest LTS version was v12.18.3.
```

Comment:
Let's update this section to use Node.js v14.x, since Node.js v16 has been released.